pages/common/cpio.md

```diff
@@ -1,16 +1,15 @@
 # cpio
 
-> Archiving utility.
+> Copies files to and from archives.
 
-- Create an archive from files
+- Take a list of file names from standard input and creates an archive from them:
 
-`echo "{{file1}} {{file2}} {{file3}}" | cpio -ov > {{archive.cpio}}`
+`echo "{{file1}} {{file2}} {{file3}}" | cpio -o > {{[o]utput_archive.cpio}}`
 
-- Create an archive from a directory
+- Create an archive containing all the files and folders in a directory in [v]erbose mode:
 
-`find {{directory}} | cpio -ov > {{archive.cpio}}`
+`find {{path/to/directory}} | cpio -ov > {{[o]utput_archive.cpio}}`
 
-- Extract an archive
-
-`cpio -idv < {{archive.cpio}}`
+- Extract files from an archive, generating [d]irectories where needed, in [v]erbose mode:
```

Comment:
I'd say "Extract all files (...)". Especially since we need a partial extraction example above this, for symmetry with the two previous examples, and to better demonstrate the core functionality of this tool (not full archiving/extraction, but partial).